openassessment/templates/openassessmentblock/staff_area/oa_staff_area.html

```diff
@@ -167,6 +167,8 @@ <h2 class="staff-info__title ui-staff__title">
         </div>
     </div>
 
+    <!-- Conditional not really required because button will be hidden. -->
```

Comment:
Hmmm, that wasn't in the commit that I cherry picked. It seems to work without it...